app/models/task.js

```diff
@@ -8,7 +8,8 @@ export default Model.extend(ContainsCodeMixin, {
   insertedAt: attr('date'),
   markdown: attr(),
   number: attr('number'),
-  position: attr('number'),
+  order: attr(),
+  position: attr(),
```

Comment:
I would set `position` below the others separately (see `stripe-platform-card`) and write that it is a virtual, write-only attribute. Would also document that `order` is read-only.